src/common/subsys_types.h

```diff
@@ -54,7 +54,7 @@ ceph_subsys_get_as_array() {
 #undef DEFAULT_SUBSYS
 }
 
-constexpr static std::uint8_t
+constexpr static uint8_t
```

Comment:
Why dropping the namespace here if one line below `std::size_t` is still being used?

I saw the bug's description but want to understand the mechanic that makes `std::size_t` is fine.

Reply:
I did some further testing, and it seems it's not necessary to drop the `std::` bit if there's sufficient `#include <cstdint>`